tawazi/node/node.py

```diff
@@ -91,8 +86,8 @@ def __init__(
         self.setup = setup
         self.unpack_to = unpack_to
 
-        self.args: List[UsageExecNode] = args or []
-        self.kwargs: Dict[Identifier, UsageExecNode] = kwargs or {}
+        self.args: List[tawazi.node.UsageExecNode] = args or []
+        self.kwargs: Dict[Identifier, tawazi.node.UsageExecNode] = kwargs or {}
```

Comment:
same here, I feel like this folder structure will encourage cyclical imports meaning a lack of flexibility